pom.xml

```diff
@@ -313,7 +313,7 @@
         <version.org.jboss.jboss-transaction-spi>7.6.0.Final</version.org.jboss.jboss-transaction-spi>
         <!-- Only used for testing -->
         <version.org.jboss.logmanager.commons-logging-jboss-logmanager>1.0.3.Final</version.org.jboss.logmanager.commons-logging-jboss-logmanager>
-        <version.org.jboss.metadata>11.0.0.Final</version.org.jboss.metadata>
+        <version.org.jboss.metadata>12.0.0.Beta1</version.org.jboss.metadata>
```

Comment:
We need final releases @ropalka

Reply:
I'll take care of that once I'm back from PTO (end of July).